assets/css/navigation.css

```diff
@@ -5,4 +5,98 @@
 
 .nav-wrapper .brand-logo {
     font-family: "KaushanScript", sans-serif;
-}
\ No newline at end of file
+}
+
+/* {
+    border:1px solid grey;
+    padding:10px;
+    height: 30px;
+} */
+
+ul li a{
+    display: flex;
+    text-align: center;
+}
+ul {
+    display: flex;
+    text-align: center;
+}
+
+ul li{
+    padding: 20px;
+}
+
+.icon{
+    padding-right: 10px;
+}
+
+.white ul a{
+    color: #000;
+}
+.white ul li.active {
+    background-color: #25adec; 
+    color:#fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li.active a{
+    background-color: #25adec; 
+    color: #fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li:hover a {
+    background-color: #73cdf7;
+    color:#fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li:focus a {
+    background-color: #73cdf7;
+    color:#fff;
+    border-radius: 5px 40px;
+}
+
+.sidenav{
+    display: inline-block;
+    text-align: center;
+    margin: auto;
+}
+
+.sidenav li{
+    text-align: center;
+    margin: auto;
+    padding: 0;
+}
+.sidenav li a{
+    display: flex;
+    text-align: center;
+    margin:auto;
+    text-decoration: none;
+}
+
+.sidenav li.active {
+    background-color: #25adec; 
+    color:#fff;
+    /* color:black; */
+    /* border:1px solid grey; */
+    border-radius: 5px 40px;
+}
+
+.sidenav li.active a{
+    background-color: #25adec; 
+    color: #fff;
+    border-radius: 5px 40px;
+}
+
+.sidenav li:hover a {
+    background-color: #73cdf7;
+    color:#fff;
+    border-radius: 5px 40px;
```

Comment:
This issue is just for the Contact Form, Do not change the style of the navbar, that can be done in another PR